_src/css/_section.scss

```diff
@@ -20,11 +26,45 @@
     .box {
       display: flex;
       flex-direction: column;
-      padding: 25px 0;
+
+      @media screen and (min-width: $hd-up) {
+        padding: 50px 0;
+      }
+
+      @media screen and (max-width: $hd-up) {
```

Comment:
@felipprodrigues same thing here. You must use `min-width: $hd` avoiding the `-up`